packages/fast-usdc/package.json

```
@@ -2,7 +2,7 @@
   "name": "fast-usdc",
```

Oh.  Didn't you mean to change this?
```suggestion
  "name": "@agoric/fast-usdc",
```

---

lol, I did mean to! https://github.com/Agoric/agoric-sdk/pull/10364/commits/139d42b5629118ba57205d5d0d72ed99b9b3aad9